gooddata-flight-server/README.md

```diff
@@ -1,3 +1,5 @@
+from typing import Anyfrom gooddata_flight_server import ServerContextfrom gooddata_flight_server import FlightDataTaskResultfrom gooddata_flight_server import TaskResultfrom typing import Union
```

Comment:
Why is this in the Readme?

Reply:
beats me.. must have accidentally pasted or something..

Reply:
fixed